File: Samples/Simple_Demo/Python_Export/Export_whole_project/parent_window.py
import os
from delphifmx import *

class Parent_Form(Form):

    def __init__(self, owner):
        self.my_button = None
        self.enter_text_edit = None
        self.enter_text_label = None
        self.main_heading = None
        self.LoadProps(os.path.join(os.path.dirname(os.path.abspath(__file__)), "parent_window.pyfmx"))

    def my_buttonClick(self, Sender):
        pass

File: Samples/Simple_Demo/parent_window.pas
unit parent_window;

interface

uses
  System.SysUtils, System.Types, System.UITypes, System.Classes, System.Variants,
  FMX.Types, FMX.Controls, FMX.Forms, FMX.Graphics, FMX.Dialogs, FMX.StdCtrls,
  FMX.Edit, FMX.Controls.Presentation;

type
  TParent_Form = class(TForm)
    my_button: TButton;
    enter_text_edit: TEdit;
    enter_text_label: TLabel;
    main_heading: TLabel;
    procedure my_buttonClick(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  Parent_Form: TParent_Form;

implementation

{$R *.fmx}

procedure TParent_Form.my_buttonClick(Sender: TObject);
begin
//
end;

end.
